cosmo_tester/test_suites/cluster/broker_management_test.py

```diff
@@ -301,6 +305,50 @@ def test_remove(brokers, logger):
     logger.info('Cluster failure recovery successful.')
 
 
+def test_remove_broker_from_manager(brokers3_and_manager, logger):
+    logger.info('Preparing cluster.')
+    brokers, manager = brokers3_and_manager[:3], brokers3_and_manager[3]
+    prepare_cluster_for_removal_tests(brokers)
+    logger.info('Cluster prepared.')
+
+    logger.info('Adding broker to manager.')
+    manager.run_command(
+        'cfy cluster brokers add {name} {ip} -n "{net}"'.format(
+            name=brokers[1].hostname,
+            ip=str(brokers[1].private_ip_address),
+            net=json.dumps({'default': str(brokers[1].private_ip_address)}),
+        )
+    )
+    logger.info('Target broker added to the manager.')
+
+    logger.info('Attempting to remove a dead node.')
+    kill_node(brokers[1])
+    get_cluster_listing(brokers, down=[brokers[1].hostname])
+    with brokers[0].ssh() as broker_ssh:
+        broker_ssh.run(
+            'cfy_manager brokers remove -r {node} 2>&1 || true'.format(
+                node=brokers[1].hostname,
+            )
+        )
+    get_cluster_listing([brokers[0], brokers[2]])
+    logger.info('Dead node removed successfully.')
+
+    logger.info('Attempting to remove a dead broker from the manager.')
+    with manager.ssh() as manager_ssh:
+        result = manager_ssh.run(
+            'cfy cluster brokers remove {node} 2>&1 || true'.format(
+                node=brokers[1].hostname)
+        ).lower()
+        assert 'removed successfully' in result
+    logger.info('Dead broker removed successfully from the manager.')
+
+    logger.info('Manager status test with one dead rabbit.')
+    with manager.ssh() as manager_ssh:
+        result = manager_ssh.run('cfy status --json 2>&1 || true')
+        assert json.loads(result.strip('\033[0m'))['status'] == 'OK'
```

Comment:
`strip('\033[0m') ` is for removing the logging.RESET_SEQ characters from the result